deploy/charts/external-secrets/templates/webhook-service.yaml

```diff
@@ -9,7 +9,7 @@ metadata:
   annotations:
     prometheus.io/path: "/metrics"
     prometheus.io/scrape: "true"
-    prometheus.io/port: {{ .Values.prometheus.service.port | quote }}
```

Comment:
Can we remove all these annotations and instead use the ServiceMonitor from above to target this service aswell?